Commands/Commands/uservoice.js

```diff
@@ -55,8 +55,8 @@ commands.vote = {
       })
     }).catch(e => {
       if (e === 'Not found') {
-        msg.reply(`I was unable to find your details, make sure you've logged into the website at <https://${config.uservoice.subdomain}.${config.uservoice.domain}> at least once.`).then(errmsg => {
-          setTimeout(() => bot.Messages.deleteMessages([msg, errmsg]), config.timeouts.errorMessageDelete)
+        msg.reply(`I was unable to find your details, make sure you've logged into the website at <https://${config.uservoice.subdomain}.${config.uservoice.domain}> at least once.`).then(() => {
+          setTimeout(() => bot.Messages.deleteMessages([msg]), config.timeouts.errorMessageDelete)
```

Comment:
This is the multiple msgs endpoint, you shouldn't be using this to delete one message.